www/docs/client/links/httpBatchStreamLink.md

```diff
@@ -0,0 +1,90 @@
+
+:::info
+We have prefixed this as `unstable_` as it's a new API, but you're safe to use it! [Read more](/docs/faq#unstable).
+:::
+
+## Usage
+
+> All usage and options are identical to [`httpBatchLink`](./httpBatchLink.md).
+
+You can import and add the `httpBatchStreamLink` to the `links` array as such:
+
+```ts title="client/index.ts"
+import { createTRPCProxyClient, httpBatchStreamLink } from '@trpc/client';
+import type { AppRouter } from '../server';
+
+const client = createTRPCProxyClient<AppRouter>({
+  links: [
+    httpBatchStreamLink({
+      url: 'http://localhost:3000',
+    }),
+  ],
+});
+```
+
+After that, you can make use of batching by setting all your procedures in a `Promise.all`. The code below will produce exactly **one** HTTP request and on the server exactly **one** database query:
+
+```ts
+const somePosts = await Promise.all([
+  trpc.post.byId.query(1),
+  trpc.post.byId.query(2),
+  trpc.post.byId.query(3),
+]);
+```
+
+## Streaming mode
+
+> ⚠️ This link is unstable and may change in the future.
+
+When batching requests together, the behavior of a regular `httpBatchStreamLink` is to wait for all requests to finish before sending the response. If you want to send responses as soon as they are ready, you can use this link. This is useful for long-running requests.
+
+```ts title="client/index.ts"
+import {
+  createTRPCProxyClient,
+  unstable_httpBatchStreamLink,
+} from '@trpc/client';
+import type { AppRouter } from '../server';
+
+const client = createTRPCProxyClient<AppRouter>({
+  links: [
+    unstable_httpBatchStreamLink({
+      url: 'http://localhost:3000',
+    }),
+  ],
+});
+```
+
+Compared to a regular `httpBatchLink`, a `unstable_httpBatchStreamLink` will:
+
+- Cause the requests to be sent with a `Trpc-Batch-Mode: stream` header
+- Cause the response to be sent with a `Transfer-Encoding: chunked` and `Vary: trpc-batch-mode` headers
+- Remove the `data` key from the argument object passed to `responseMeta` (because with a streamed response, the headers are sent before the data is available)
+
+## Compatibility (client-side)
+
+Browser support should be identical to [`fetch`](https://caniuse.com/fetch) support.
+
+For runtimes other than the browser ones, the `fetch` implementation should support streaming, meaning that the response obtained by `await fetch(...)` should have a `body` property of type `ReadableStream<Uint8Array> | NodeJS.ReadableStream`, meaning that:
+
+- either `response.body.getReader` is a function that returns a `ReadableStreamDefaultReader<Uint8Array>` object
+- or `response.body` is a `Uint8Array` `Buffer`
+
+This includes support for `undici`, `node-fetch`, native Node.js fetch implementation, and WebAPI fetch implementation (browsers).
+
+## Compatibility (server-side)
+
+> ⚠️ for **aws lambda**, `unstable_httpBatchStreamLink` is not supported (will simply behave like a regular `httpBatchStreamLink`). It should not break anything if enabled, but will not have any effect.
```

Comment:
Same as above, we're actually referring to `httpBatchLink` here because for aws lambda, both links behave the same.

```suggestion
> ⚠️ for **aws lambda**, `unstable_httpBatchStreamLink` is not supported (will simply behave like a regular `httpBatchLink`). It should not break anything if enabled, but will not have any effect.
```